packages/@ember/deprecated-features/deprecated-features.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-implicit-coercion */
```

Comment:
Lets move this to be the `packages/@ember/deprecated-features/index.js` (along with the `packages/@ember/deprecated-features/deprecated-features.d.ts` -> `packages/@ember/deprecated-features/index.d.ts`).